packages/client/lib/net/protocol/ethprotocol.ts

```diff
@@ -136,7 +136,7 @@ export class EthProtocol extends Protocol {
   encodeStatus(): any {
     // TODO: add latestBlock for more precise ETH/64 forkhash switch
     return {
-      networkId: this.chain.networkId,
+      networkId: this.chain.networkId.toNumber(),
```

Comment:
This change is currently breaking the client CLI start on Yolo v3.

Reply:
Just checked, for RLPx the `networkId` can actually be removed completely from here, since the `devp2p` library is not using this passed in `networkId` any more but [using the networkId from Common directly](https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/devp2p/src/eth/index.ts#L197).

Not sure about libp2p though.